File: VirtualTryonWomenFashion/VirtualTryonWomenFashion.Service/DTO/Transaction/TransactionInformation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VirtualTryonWomenFashion.Service.DTO.Transaction
{
    public class TransactionInformation
    {
        public int TransactionId { get; set; }

        public int UserId { get; set; }

        public int OrderId { get; set; }

        public string Status { get; set; }

        public decimal? Money { get; set; }

        public string Method { get; set; }

        public string TransactionCode { get; set; }

        public DateTime UpdatedAt { get; set; }

        public DateTime CreatedAt { get; set; }
    }
}

File: virtual-fashion-for-women-FE/src/models/TransactionInformation.ts
export interface TransactionInformation {
    transactionId: number;
    userId: number;
    orderId?: number;
    status: string;
    money: number;
    method: string;
    type: string;
    transactionCode: string;
    bankName?: string;
    bankAccountNumber?: string;
    createdAt: string;
    updatedAt: string;
}